compiler/modules/front/codeGen.bal

```diff
@@ -991,7 +1000,7 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
             return { result, block: nextBlock, narrowing };
         }
         var { checkingKeyword, operand } => {
-            return codeGenCheckingExpr(cx, bb, env, checkingKeyword, operand);
+            return <CodeGenError|RegExprEffect> codeGenCheckingExpr(cx, bb, env, checkingKeyword, operand, false);
```

Comment:
Possible refactoring: rename current `codeGenCheckingExpr` to `codeGenChecking`, have separate `codeGenCheckingExpr() returns RegExprEffect` and `codeGenCheckingStmt() returns RegExprEffect|StmtEffect`, and move above cast to former.